ngm/__init__.py

```diff
@@ -95,11 +95,22 @@ def dominant_eigen(X: np.ndarray, norm: str = "L1") -> DominantEigen:
     value = e.eigenvalues[i]
     vector = _ensure_positive_array(e.eigenvectors[:, i])
 
+    # check for negative values
     if not value > 0:
         raise RuntimeError(f"Negative dominant eigenvalue: {value}")
     if not all(vector >= 0):
         raise RuntimeError(f"Negative dominant eigenvector values: {vector}")
 
+    # check for complex numbers
+    if np.iscomplex(value) and value.imag != 0:
+        raise RuntimeError("Complex numbers in eigenvalue")
```

Comment:
```suggestion
    if not np.isreal(value):
        raise RuntimeError("Complex eigenvalue")
```
`np.isreal` works on arrays with both real and complex *types*. See the first example in https://numpy.org/doc/2.1/reference/generated/numpy.isreal.html